doc/developers/testsuite.html

```diff
@@ -793,13 +793,12 @@
                          directory (without actually running the
                          testsuite)
 
-    "Nightly"          - Reserved for nightly regression tests for
-                         build bots on various architectures
+    "Regression Tests" - Reserved for the "official" regression tester
 
-    "Regression Tests" - Reserved for the regression tester
+    "Continuous"       - Reserved for the "official" regression tester
```

Comment:
can you elaborate the difference between `Continuous` and `Regression Tests`? Copy-pasting the description does not help much :smile:

Reply:
@tjhei 

> I don't know what the precise definition of the different tracks is, but I created #5239 . Maybe Matthias can tell us what they mean.

just saw that in gitter. 

@tamiko ping.

Reply:
@tamiko ping.

Reply:
Currently, `tester` submits to `Regression Tests` (which is now located in Minneapolis) and `simserv04` submits to `Continuous` (which is in Heidelberg).

The choice for tracks is somewhat arbitrary.

Reply:
@tamiko so they both run on `cron` every that many hours or they test each commit? Is there any difference at all between the two setups? 

> The choice for tracks is somewhat arbitrary.

maybe use one track then for both and document here that, say, Regression tests are official tester setup and maintained by Mathias Maier, whereas Experimental is the track for all other test results submitted by other developers?